webCompression.ts: 'brotli' not assignable to CompressionFormat in DOM lib

## Problem

`keryx/util/webCompression.ts` line 116 uses `"brotli"` as a `CompressionFormat`, but the DOM lib's `CompressionFormat` type only includes `"deflate" | "deflate-raw" | "gzip"`. This causes a type error when consumers type-check keryx source with DOM lib types (e.g., frontend projects using `@backend/*` path alias):

```
keryx/util/webCompression.ts(116,42): error TS2345: Argument of type '"gzip" | "brotli"' is not assignable to parameter of type 'CompressionFormat'.
  Type '"brotli"' is not assignable to type 'CompressionFormat'.
```

## Context

- Keryx ships `.ts` source files, so `skipLibCheck: true` doesn't apply (only skips `.d.ts`)
- Bun's type definitions include `"brotli"` in `CompressionFormat`, but DOM lib does not
- This surfaces when a frontend `tsconfig.app.json` with `"lib": ["ES2022", "DOM"]` transitively imports keryx via path-aliased backend actions

## Suggested Fix

Either:
1. Cast to `any`/`as CompressionFormat` in `webCompression.ts` to suppress the error
2. Augment `CompressionFormat` in keryx's own type declarations
3. Ship compiled `.js` + `.d.ts` instead of raw `.ts` source

## Workaround

Removed `tsc -b` from frontend build script, relying on Vite (esbuild) for type stripping.

--- a/packages/keryx/package.json
+++ b/packages/keryx/package.json
@@ -1,6 +1,6 @@
 {
   "name": "keryx",
-  "version": "0.14.1",
+  "version": "0.14.2",
   "module": "index.ts",
   "type": "module",
   "license": "MIT",

--- a/packages/keryx/util/webCompression.ts
+++ b/packages/keryx/util/webCompression.ts
@@ -111,9 +111,8 @@ export async function compressResponse(
     // Compress the buffered body
     const format: Bun.CompressionFormat = encoding === "br" ? "brotli" : "gzip";
     // @ts-ignore Bun supports "brotli" as CompressionFormat but DOM lib does not
-    const stream = new Blob([body])
-      .stream()
-      .pipeThrough(new CompressionStream(format));
+    const compressionStream = new CompressionStream(format);
+    const stream = new Blob([body]).stream().pipeThrough(compressionStream);
 
     const headers = new Headers(response.headers);
     headers.set("Content-Encoding", encoding);
@@ -129,7 +128,8 @@ export async function compressResponse(
   // Content-Length is present and above threshold — stream-compress
   const format: Bun.CompressionFormat = encoding === "br" ? "brotli" : "gzip";
   // @ts-ignore Bun supports "brotli" as CompressionFormat but DOM lib does not
-  const stream = response.body.pipeThrough(new CompressionStream(format));
+  const compressionStream = new CompressionStream(format);
+  const stream = response.body.pipeThrough(compressionStream);
 
   const headers = new Headers(response.headers);
   headers.set("Content-Encoding", encoding);

